apps/test/unit/templates/manageStudents/manageStudentsReduxTest.js

```diff
@@ -51,6 +53,48 @@ const studentEmailData = {
     },
 };
 
+const studentPictureData = {
+  1: {
+      id: 1,
+      name: 'StudentNameA',
+      username: 'student1',
+      userType: 'student',
+      age: 17,
+      gender: 'f',
+      loginType: 'picture',
```

Comment:
tiny ? : are picture logins mostly for younger kids?

Reply:
younger students definitely use it almost exclusively, but some ms/hs teachers prefer it because its easier for students to manage